packages/prosemirror-lwdita-backend/src/api/controller/github.controller.ts

```diff
@@ -33,6 +33,9 @@ export const authenticateUserWithOctokit = async (req: Request, res: Response) =
   // dynamically import the module to avoid EMS and CJS incompatibility
   const { authenticateWithOAuth } = await import('../modules/octokit.module.mjs');
 
+  console.log("code", code);
```

Comment:
Is this log still needed?

Reply:
No, Fixed in 625fd3a3319e6d0da09f769cda2e88480cfa4769